lib/mayaUsd/undo/OpUndoItemList.cpp

```diff
@@ -33,12 +33,24 @@ struct OnSceneResetListener : public PXR_NS::TfWeakBase
     {
         PXR_NS::TfWeakPtr<OnSceneResetListener> self(this);
         PXR_NS::TfNotice::Register(self, &OnSceneResetListener::OnSceneReset);
+        PXR_NS::TfNotice::Register(self, &OnSceneResetListener::OnMayaAboutToExit);
+        PXR_NS::TfNotice::Register(self, &OnSceneResetListener::OnMayaAboutToResetScene);
```

Comment:
Combining this registration into one call made sense to me in your original change.

Reply:
Yeah, OTOH, I can imagine some case where they would only need to do something on exit, so having them separate is not a big cost and provides the flexibility.